__tests__/app.test.js

```diff
@@ -299,3 +299,58 @@ describe('POST /api/articles/:article_id/comments', () => {
     
 });
 
+describe('PATCH /api/articles/:article_id', () => {
+    it('responds with updated article', () => {
+        const articleId = 1;
+        const incrementVotes = 5;
+        let initialVotes;
+      
+        return request(app)
+          .get(`/api/articles/${articleId}`)
+          .expect(200)
+          .then((res) => {
+            initialVotes = res.body.article.votes;
+            return request(app)
+              .patch(`/api/articles/${articleId}`)
+              .send({ inc_votes: incrementVotes })
+              .expect(200);
+          })
+          .then((res) => {
+            const updatedVotes = res.body.article.votes;
+            expect(updatedVotes).toBe(initialVotes + incrementVotes);
```

Comment:
We also need to check you're getting the whole object back, not just the updated votes

Reply:
new test checks we're getting the whole object back